### PR TITLE
OSDOCS-3136: Update Dedicated and ROSA troubleshooting content

### PR DESCRIPTION
**Pull request by bmcelvee**
Version(s):
4.11, 4.12

Issue:
https://issues.redhat.com/browse/OSDOCS-3136

Link to docs preview:
Dedicated: https://50896--docspreview.netlify.app/openshift-dedicated/latest/osd_support/osd-managed-resources.html

ROSA: https://50896--docspreview.netlify.app/openshift-rosa/latest/rosa_support/rosa-managed-resources.html

QE review:
This is existing support content, no QE review required. 

Additional information:

### CODE REVIEW

**Comment by bmcelvee** [comment]
@arendej Please give this PR a review when you get a chance. Thanks!

**Review by arendej — APPROVED** [review]
LGTM

File: osd_support/osd-managed-resources.adoc [hunk 1]
```diff
@@ -28,7 +28,7 @@ include::https://raw.githubusercontent.com/openshift/managed-cluster-config/mast
 [id="osd-add-on-managed-namespaces"]
 == {product-title} add-on namespaces
 
-{product-title} add-ons are services available for installation after cluster installation. These additional services include AWS CloudWatch, Red Hat CodeReady Workspaces, Red Hat OpenShift API Management, and Cluster Logging Operator. Any changes to resources within the following namespaces might be overridden by the add-on during upgrades, which can lead to unsupported configurations for the add-on functionality.
+{product-title} add-ons are services available for installation after cluster installation. These additional services include Red Hat CodeReady Workspaces, Red Hat OpenShift API Management, and Cluster Logging Operator. Any changes to resources within the following namespaces might be overridden by the add-on during upgrades, which can lead to unsupported configurations for the add-on functionality.
```
Comment (bscott-rh) [inline osd_support/osd-managed-resources.adoc:31]: FYI, CodeReady Workspaces is called "Red Hat OpenShift Dev Spaces" now. You can use `{openshift-dev-spaces-productname}` if you also include `_attributes/common-attributes.adoc`.

File: rosa_support/rosa-managed-resources.adoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+[source,yaml]
+----
+include::https://raw.githubusercontent.com/openshift/managed-cluster-config/master/resources/managed/all-osd-resources.yaml[]
+----
+====
+
+[id="rosa-add-on-managed-namespaces"]
+== {product-title} add-on namespaces
+
+{product-title} add-ons are services available for installation after cluster installation. These additional services include Red Hat CodeReady Workspaces, Red Hat OpenShift API Management, and Cluster Logging Operator. Any changes to resources within the following namespaces might be overridden by the add-on during upgrades, which can lead to unsupported configurations for the add-on functionality.
```
[the remark this answers is not in the document]
Comment (bscott-rh) [reply rosa_support/rosa-managed-resources.adoc:31]:
```suggestion
{product-title} add-ons are services available for installation after cluster installation. These additional services include Red Hat CodeReady Workspaces, Red Hat OpenShift API Management, and Cluster Logging Operator. Any changes to resources within the following namespaces can be overridden by the add-on during upgrades, which can lead to unsupported configurations for the add-on functionality.
```
Tiny wording suggestion.

File: rosa_support/rosa-managed-resources.adoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+[source,yaml]
+----
+include::https://raw.githubusercontent.com/openshift/managed-cluster-config/master/resources/addons-namespaces/main.yaml[]
+----
+====
+
+[id="rosa-validating-webhooks"]
+== {product-title} validating webhooks
+
+{product-title} validating webhooks are a set of dynamic admission controls maintained by the OpenShift SRE team. These HTTP callbacks, also known as webhooks, are called for various types of requests to ensure cluster stability. Upon request the webhooks accept or reject the request. The following list describes the various webhooks with rules containing the registered operations and resources that are controlled. Any attempt to circumvent these validating webhooks could affect the stability and supportability of the cluster.
```
Comment (bscott-rh) [inline rosa_support/rosa-managed-resources.adoc:45]:
```suggestion
{product-title} validating webhooks are a set of dynamic admission controls maintained by the OpenShift SRE team. These HTTP callbacks, also known as webhooks, are called for various types of requests to ensure cluster stability. The following list describes the various webhooks with rules containing the registered operations and resources that are controlled. Any attempt to circumvent these validating webhooks could affect the stability and supportability of the cluster.
```
Minimalism suggestion

**Comment by bscott-rh** [comment]
Left a couple of small suggestions. LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Pull request merged**